doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_memoryr.rst

```diff
@@ -1,5 +1,3 @@
-:orphan:
-
 .. _ug_nrf54h20_architecture_memory:
```

Comment:
@hubertmis Hey Hubert, could you leave here the comments regarding the updates needed to the memory layout?